components/modules/AdminDashboard/_modules/BestArtist/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { songs } from "../../../utils/dummy";
+import numberConverter from "../../../utils/helpers/numberConverter";
+import ArtistCard from "../Cards/ArtistCard";
+
+const BestArtist = () => {
+  return (
+    <div className="w-full h-[440px] bg-white p-5 shadow-xl rounded-xl">
+      <h3 className="text-2xl font-semibold pb-5 border-b">Best Artist</h3>
+      <div className="flex flex-col gap-3">
```

Comment:
great work but kindly apply the changes bellow to make it more cleaner
create some dummy data that you can map through to display the best artists please!
```suggestion
      <div className="flex flex-col gap-3">
```
```suggestion
      <div className="flex flex-col gap-3">
```